<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgot_dash_password</class>
 <widget class="QDialog" name="Forgot_dash_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Savior : Forgot Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/src.qrc">
    <normaloff>:/pic/Icon.png</normaloff>:/pic/Icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;; color: #ff0000;</string>
     </property>
     <property name="text">
      <string>Did you forgot your password ?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff0000;&quot;&gt;If after click on the &amp;quot;Rest&amp;quot; button no confirmation&lt;br/&gt;message pops up. It means your password have &lt;br/&gt;not been rested!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_nPwd">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;; color: #0000ff;</string>
           </property>
           <property name="text">
            <string>New Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_cPwd">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;; color: #0000ff;</string>
           </property>
           <property name="text">
            <string>Confirm Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_email">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;; color: #0000ff;</string>
           </property>
           <property name="text">
            <string>Email   *</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="line_nPwd">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_cPwd">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_email"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Rest">
       <property name="styleSheet">
        <string notr="true">color: #0000ff;</string>
       </property>
       <property name="text">
        <string>Rest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="styleSheet">
        <string notr="true">color: #0000ff;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_nPwd</tabstop>
  <tabstop>line_cPwd</tabstop>
  <tabstop>line_email</tabstop>
  <tabstop>Rest</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="Resource/src.qrc"/>
 </resources>
 <connections/>
</ui>
